lexicons/app/bsky/feed/postgate.json

```diff
@@ -16,16 +16,16 @@
             "format": "at-uri",
             "description": "Reference (AT-URI) to the post record."
           },
-          "detachedQuotes": {
+          "detachedEmbeddingUris": {
```

Comment:
While we're in here I wonder if we may want to sync-up the terminology on embed views.  Currently there's a `#viewRemoved` lexicon that gets used within the posts listed on this field.  For consistency perhaps that should become `#viewDetached`.